assets/css/styles.css

```diff
@@ -630,29 +660,115 @@ header .display-mobile img {
   display: none;
 }
 
+/*
+HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
+*/
+
+.hamburger-navmenu li:last-child {
+  border-bottom: none;
+}
+
+.hamburger-navmenu li a:hover {
+  font-weight: bold;
+}
+
 @media (max-width: 280px) {
-  .project-built-with li,
-  .list-competences li {
+  .header-nav .display-mobile {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+  }
+
+  .header-nav .nav-menu {
+    display: none;
+  }
+
+  .header-nav .hamburger-menu {
+    height: 5vh;
+    margin-top: 0%;
+  }
+
+  .project .project-built-with li,
+  .competences-card .list-competences li {
     width: 90%;
     text-align: center;
   }
+
+  footer .social .display-mobile,
+  .header .social .display-mobile {
+    width: 30%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .display-desktop {
+    display: none;
+  }
 }
 
 @media (min-width: 375px) {
   .display-desktop {
     display: none;
   }
 
-  header .display-mobile {
+  .header-nav .display-mobile {
     width: 100%;
     display: flex;
     justify-content: flex-end;
     align-items: flex-end;
   }
```

Comment:
- [ ] Good Job 👏 so far but you still need to improve on some things, I noticed a discrepancy between the side menu in your design and the design in the Figma mockup. I understand that it can be tempting to deviate from the design when building a project; however, I would recommend that you adhere to the design as closely as possible to ensure a professional and polished final product.
     - Figma design:
       ![firefox_2mMVkGRmce](https://user-images.githubusercontent.com/104968236/231766464-74203787-42ec-43df-8a7b-e0e7d7ec4072.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/231766677-3b6b9bcf-628b-4946-9b4a-01f284c45ba3.png)
- [ ] Please make sure you follow the proper background color, font color, font size, and Icons for the cross as specified in the design
- [ ] Also kindly fix the height on your design as it should almost fill the screen with a small space in the bottom Kindly check the design again.

Reply:
Done @AbelG101